src/setup-dotnet.ts

```diff
@@ -8,7 +9,20 @@ async function run() {
     // Version is optional.  If supplied, install / use from the tool cache
     // If not supplied then task is still used to setup proxy, auth, etc...
     //
-    const version = core.getInput('version');
+    let version = core.getInput('version');
+    if (!version) {
+      // Try to fall back to global.json
+      const globalJsonPath = path.join(process.cwd(), 'global.json');
```

Comment:
Will this always be at the root or do we need to specify a way to give a relative path?

Reply:
This should be at the root. If for some reason its not, we can change the cwd using `working-directory`

Reply:
@damccorm does `working-directory` work with actions? When I mixed `uses` and `working-directory` my workflow wouldn't run but when I used it with `run` it did so I assumed that was for scripts only.

Reply:
I think it doesn't yet, but it will. I'd rather wait on that than put in something here that won't be necessary when that's implemented